src/client/components/Login.jsx

```diff
@@ -1,16 +1,79 @@
+import { useState } from "react";
+
+const emptyDetails = {
+    username: "",
+    password: ""
+}
+
+const loginURL = "http://localhost:4000/user/login"; //need to update this URL when we try to run the fetch.
```

Comment:
again, need to update this route so users cannot access it when we dont want them to